packer/wowza/provisioning.json

```diff
@@ -0,0 +1,28 @@
+{
+    "variables": {
+        "wowza_ami": "ami-91960ce2"
+    },
+    "builders": [{
+        "type": "amazon-ebs",
+        "region": "eu-west-1",
+        "source_ami": "{{user `wowza_ami`}}",
+        "instance_type": "c3.large",
```

Comment:
I think we need c3.xlarge to do transcoding, it lags a bit on the large we're testing with at the moment